apps/src/JSInterpreter.js

```diff
@@ -104,6 +105,9 @@ var JSInterpreter = module.exports = function (options) {
   };
 
   try {
+    // Return value will be stored as this.interpreter inside the supplied
+    // initFunc() (other code in initFunc() depends on this.interpreter, so
+    // we can't wait until the constructor returns)
```

Comment:
Now that we are hooking `getProperty` and `setProperty`, those hooked versions are getting called during the init sequence (in a few places), and those all expect `this.interpreter` to be set.